ss_admin_console/main/default/classes/utils/SSACUtil.cls

```diff
@@ -16,9 +16,14 @@ public inherited sharing class SSACUtil {
         List<Type> result = new List<Type>();
         List<ApexClass> classes = getApexClasses();
         for (ApexClass aClass : classes) {
-            Type classType = Type.forName(aClass.Name);
-            if (classType != null && theType.isAssignableFrom(classType)) {
-                result.add(classType);
+            try {
+                Type classType = Type.forName(aClass.Name);
+                if (classType != null && theType.isAssignableFrom(classType) && theType != classType) {
+                    result.add(classType);
+                }
+            }
+            catch (Exception e) {
+                throw e;
```

Comment:
What is the idea of rethrowing the exception?

Reply:
"throw e;" replaced with "System.debug('Error in SSACUtil.getSubClasses: ' + e.getMessage());"